OneCommunityGlobal/HighestGoodNetworkApp
# Ivy-bar-chart-summary-and-barChart-template

# Description
Please include the exact bug/functionality description and a summary of the changes/ related issues. Please also include any other relevant motivation and context:

## Main changes explained:
- Rename barChart.js for template…
- Create HoursCompletedBarChart.js for introducing new components …
…

## How to test:
1. check into current branch
2. do `npm install` and `...` to run this PR locally
3. Clear site data/cache
4. go to `http://localhost:3000/totalorgsummary`, to check the Hours Completed bar, the start date and end date, last start date and last end date will replaced by date filter later. Here is just an example.

## Screenshots or videos of changes:
<img width="419" alt="Screenshot 2024-08-03 at 1 26 09 PM" src="https://github.com/user-attachments/assets/23ee1d7b-e37f-43ff-b769-4cfe07c454e1">

Include the information the reviewers need to know.
